vertical-pod-autoscaler/pkg/recommender/routines/capping_post_processor.go

```diff
@@ -17,10 +17,10 @@ limitations under the License.
 package routines
 
 import (
+	"k8s.io/klog/v2"
```

Comment:
this is not an add, just lastest version of linter making different organisation of imports.
Same as https://github.com/kubernetes/autoscaler/pull/5676/files/143402350b219a8ee36ea451ec2f5535a097ddcf#r1173813414